client/src/components/composite/SignUpForm/Sections/ContactSection.tsx

```diff
@@ -1,32 +1,45 @@
-import { TextField } from "@mui/material"
+import { useSignUpFormData } from "store/signupform"
+import TextInput from "components/generic/TextInputComponent/TextInput"
 
 export const ContactSection = () => {
+  const [
+    { emergency_name, emergency_relation, emergency_phone },
```

Comment:
Not sure how I should update the state of the inputs that hold the email & phone number, as I can't find these in the UserAdditionalInfo schema?

Reply:
hm, I think I will merge these all into one field to avoid complexity